pypuck/pypuck.py

```diff
@@ -198,12 +201,16 @@ def team_stats(start_season="20192020", end_season="20192020"):
     The valid seasons range from the 1917 season until the
     current season.
 
+    You can find the glossary pertaining to the returned
+    columns by going to http://www.nhl.com/stats/glossary.
+
     Parameters
     ----------
       start_season : str
         The stat start year string in 'YYYYYYYY' format.
       end_season : str
         The stat end year string in 'YYYYYYYY' format.
+    
```

Comment:
```suggestion
```

@ManishPJoshi  You've added a blank line which is causing our build to fail because it does not comply with flake8 formatting. Please remove this blank line and see that the tests pass.